docs/source/model_design/risk_factors.rst

```diff
@@ -200,6 +200,109 @@ Confounding
+****************************************
+
+Now, let's imagine that a brand new hypothetical medication was just 
+created that dissolves tar in lungs that accumulates due to smoking and is now 
+widely used in some areas. Our causal diagram would now look like this:
+
+.. image:: medication_example.svg
+
+Quickly, we can now see that the relationship we previously knew between 
+smoking and lung cancer is now impacted by the use of this hypothetical 
+medication that affects the intermediate variable between smoking and lung 
+cancer. Therefore, now we can only expect our intervention to decrease 
+population lung cancer rates only in areas that do not widely use this 
+medication.
+
+Notably, intermediate variables may be relevant in situations in which the 
+variable located most proximal to the outcome in the causal pathway is 
+difficult to measure. For instance, measuring the amount of tar in a person's 
+lungs is an invasive procedure; measuring the amount of cigarettes a person 
+smokes is much easier in comparison. Therefore, data availability may dictate 
+that we model cigarette smoking rather than lung tar. 
+
+  This may be a reasonable approach, given that the intermediate variable 
+  *does not* interfere with the relationship between the exposure and outcome 
+  for the given model population (little or no use of the medication that 
+  removes tar from lung). However, if there *is* interference by the 
+  intermediate variable on the relationship between the exposure and outcome 
+  (significant use of the medication), then the previously measured 
+  relationship between smoking and lung cancer will not apply to this 
+  population in the same way.
+
+Now, let's say that some time has gone by and now this medication has become 
+quite common. A new study measured the relationship between smoking and lung 
+cancer in a study population in which 50% of smokers used the medication. Now, 
+let's say that we are interested in using the data from that study on the 
+relationship between smoking and lung cancer in a simulation for a different 
+location. However, in the location we wish to model, the medication is not 
+approved at a national level and therefore use of the medication is close to 
+zero. However, since we know that the relationship between smoking and lung 
+cancer as we've defined it in this example is affected by the use of this 
+medication, applying the data from this new study to this model location would 
+be inappropriate. Rather, we should use data that measured the relationship 
+between smoking and lung cancer in a study population with a similar exposure 
+to the medication intermediate variable.
+
+Additional Considerations
+*************************
+
+Another example of when an intermediate variable might interfere with the 
+relationship between an exposure and outcome is when a given intervention 
+*decreases* the prevalence of an intermediate variable (that is related to an 
```

Comment:
isnt this the medication that dissolves tar example?

Reply:
Yeah, I guess I was thinking that this example could also be that there is just no baseline burden of the intermediate variable (like no malaria or zika exposure because of climate or something along those lines). Do you think this should be clarified and/or that it is worthwhile to include?

Reply:
(See my response in the comment thread below)